tests/libvmi/vmi.go

```diff
@@ -20,6 +20,7 @@
 package libvmi
 
 import (
+	"github.com/onsi/ginkgo/v2"
```

Comment:
I think it's a great idea to annotate all test VMs, but I don't think that it is acceptable for `libvmi` to depend on the framework that uses it. Can we find a way to inject this annotation without adding this dependency?

Reply:
Agreed. Pushed v2 without the dependency, PTAL!

Reply:
How about we create infra to register default options by users at `init()`?

Reply:
```go
func init() {
  libvmi.RegisterDefaultOptions(withTestAnnotation)
}
```

Reply:
> ```go
> func init() {
>   libvmi.RegisterDefaultOption(withTestAnnotation)
> }
> ```

Pushed something like that, PTAL